### ISSUE TITLE
Code Table Request - new examined_detected term 

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6076]
## Instructions

_This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions._

_Reviewing documentation before proceeding will result in a more enjoyable experience._

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

See https://github.com/ArctosDB/arctos/issues/6042#issuecomment-1487659013

_**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

`Virus: Orthohantavirus`

_**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

`Orthohantavirus is a genus of single-stranded, enveloped, [negative-sense](https://en.wikipedia.org/wiki/Negative-sense) [RNA viruses](https://en.wikipedia.org/wiki/RNA_virus) in the family [Hantaviridae](https://en.wikipedia.org/wiki/Hantaviridae) within the order [Bunyavirales](https://en.wikipedia.org/wiki/Bunyavirales).[[3]](https://en.wikipedia.org/wiki/Orthohantavirus#cite_note-ECDC-3) Members of this genus may be called orthohantaviruses or simply hantaviruses. [Wikipedia](https://en.wikipedia.org/wiki/Orthohantavirus)`

_**Context**: Describe why this new value is necessary and existing values are not._

To appropriately track exam and detection from various projects

_**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

ctexamined_detected (to be created)

_**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

_**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions._

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
The first term in a new code table will set the tone for those that follow!

1. Will the colon cause problems?
2. Are we sticking with Proper Case?
3. Is it appropriate to go with genus level here (how many Virus: Genus terms will end up in the table?) or should the details go into remark? I know we looked at this today, but I want everyone to understand future implications and to consider code table fatigue when the list becomes too long. See also the "[endless dropdown of death](https://github.com/ArctosDB/arctos/issues/6067#issuecomment-1487522088)."

**Comment by dustymc** [comment]
>Will the colon cause problems?

I don't think so, it's not a 'natural separator' in any relevant language, not easily confused with other characters, seems OK.

>Are we sticking with Proper Case?

No, all of Arctos is lowercase except when there's a proper noun involved. I think this should be ``virus: Orthohantavirus`` (unless I'm misunderstanding something)

> appropriate to go with genus level 

No idea! Clearly the data start degrading at some number of options. 10 is certainly safe, 100 is _almost_ certainly attracting entropy. (And maybe the general: specific pattern that's been suggested extends that to 10 groups rather than items??) Is anything of great importance lost by moving up a rung to 'hantavirus'? If not then fewer==better, if so then use what's needed.

**Comment by Jegelewicz** [comment]
@ArctosDB/arctos-code-table-administrators please review.

**Comment by Jegelewicz** [comment]
@jldunnum @bryansmclean @campmlc @gracz-UNL @ehalverson26 @adhornsby

**Comment by campmlc** [comment]
@jldunnum said that was the new term for hantavirus in general, although I did propose going with the least specific value. I don't believe this is a genus, but maybe a family? If so, maybe we can ditch the caps? @jldunnum

**Comment by jldunnum** [comment]
It is the new genus name, Hantavirus is a synonym. See the International Committee on Taxonomy of viruses.
https://ictv.global/taxonomy/taxondetails?taxnode_id=20180020

______________________________________________________________
Jonathan L. Dunnum Ph.D. (he, him, his)
Senior Collection Manager
Division of Mammals, Museum of Southwestern Biology
University of New Mexico
Albuquerque, NM 87131
(505) 277-9262
Fax (505) 277-1351

Chair, Systematic Collections Committee, American Society of Mammalogists
Latin American Fellowship Committee, ASM

MSB Mammals website: http://www.msb.unm.edu/mammals/index.html
Facebook: http://www.facebook.com/MSBDivisionofMammals

Shipping Address:
Museum of Southwestern Biology
Division of Mammals
University of New Mexico
CERIA Bldg 83, Room 204
Albuquerque, NM 87131

**Comment by dustymc** [comment]
This is my current blocker. ``virus: Orthohantavirus`` still seems correct from here. Help?

**Comment by campmlc** [comment]
That works!

**Comment by campmlc** [comment]
change to virus: Orthohantavirus.  @jldunnum please OK.

**Comment by jldunnum** [comment]
Ok by me!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dustymc** [comment]
Is there some problem with this? It still hasn't been reviewed, I have a timeslot in which I was hoping to finish this up NOW. ???

**Comment by campmlc** [comment]
I checked it last week. There are too many of these to track. @Jegelewicz

**Comment by dustymc** [comment]
>checked it last week

Nope....

<img width="392" alt="Screenshot 2023-04-04 at 8 46 39 AM" src="https://user-images.githubusercontent.com/5720791/229846662-93b4d310-cd56-4c2d-b74f-7e99cce9eaef.png">
